app/lib/main.dart

```diff
@@ -1,4 +1,9 @@
+import 'package:firebase_core/firebase_core.dart';
 import 'package:flutter/material.dart';
 import 'humane_transport_app.dart';
 
-void main() => runApp(HumaneTransportApp());
+Future<void> main() async {
```

Comment:
I've not heard of having an `async` main. What does `Firebase.initializeApp()` do that we need it, and;
- [ ] does it wipe user data on close? We want to cache ATRs.

Reply:
This is required to initialize Firebase to Flutter. You won't be able to use firebase without doing so

Reply:
What does this mean? Does the Firebase "store" on the phone disconnect from the app when the app is closed?

- [ ] We should do something if we can't connect

Reply:
There has to be another way to initialize firebase to our app, but at the moment the tutorial I was following uses this method. At this point, I can't really find what's the best practice for initializing firebase. Everyone seems to have their own approach. If we use this approach, then you are correct! We should definitely do something if user can't connect.